tools/proto-compiler/src/constants.rs

```diff
@@ -98,7 +99,7 @@ pub static CUSTOM_TYPE_ATTRIBUTES: &[(&str, &str)] = &[
     (".tendermint.types.Commit", SERIALIZED),
     (".tendermint.types.CommitSig", SERIALIZED),
     (".tendermint.types.ValidatorSet", SERIALIZED),
-    (".tendermint.crypto.PublicKey", SERIALIZED),
+    (".tendermint.crypto.PublicKey.sum", SERIALIZED),
```

Comment:
Is that a typo, or does Prost expect the type name in lowercase? I am asking because that `#[derive]` argument only applies to structs/enums AFAIK. 

```suggestion
    (".tendermint.crypto.PublicKey.Sum", SERIALIZED),
```

Reply:
In any case, it does not seem to be reflected in the generated protos.

Reply:
It actually is reflected and the name must be in lowercase. Without this, the `Sum` type does not get the serde attributes, which it previously "inherited" from the configuration for the containing message. This is how it works with prost-build.

Reply:
Gotcha!